### PR TITLE
Adding three sections of tests

### PR DESCRIPTION
**Pull request by edjuaro**
As per discussed on Slack on 2018-04-09

### CODE REVIEW

File: buildsuites.xml [hunk 1]
```diff
@@ -271,7 +275,7 @@
             <resources refid="ShortHeatMapViewerTests" />
             <resources refid="ShortHierarchicalClusteringTests" />
             <resources refid="ShortTopHatTests" />
-            <resources refid="Shorttxt2odfTests" />
+            <!-- <resources refid="Shorttxt2odfTests" /> --> <!-- Removing this from IU. Assuming it is not installed there either -->
```
Comment (peterjcarr) [inline buildsuites.xml]: I was under the impression that we want to add "txt2odf" to the "short-nightly-tests-suite".

File: buildsuites.xml [hunk 2]
```diff
@@ -288,6 +292,38 @@
...
+            <!-- <resources refid="Shorttxt2odfTests" /> --> <!-- Removing this from Broad, we are not putting new modules on old servers-->
+            <resources refid="nightlyMergeHTSeqCountsTests" />
+            <resources refid="nightlyPreprocessDatasetTests" />
+            <resources refid="nightlyPreprocessReadCountsTests" />
+        </union>
+        <!-- The GpUnit code parses filesets using the value returned by a call to File.pathSeparator() -->
+    	<!-- as the delimiter. This returns ";" on Windows and ":" on Linux. However, ant creates filesets -->
+    	<!-- with ";" as the separator on both platforms (!). The pathconvert task transforms the fileset -->
+    	<!-- to one with a delimiter that properly matches what the code expects. -->
+        <pathconvert property="short.nightly.tests" refid="short.tests" />
```
Comment (peterjcarr) [inline buildsuites.xml]: is this the correct refid?

File: buildsuites.xml (hunk 1 above)
Comment (edjuaro) [reply buildsuites.xml]: Yeah, I added them to short-nightly-tests but removed them from short-nightly-tests-iu

File: buildsuites.xml (hunk 2 above)
Comment (edjuaro) [reply buildsuites.xml]: I think this is an overlook on my end. line 314 should be:
`<pathconvert property="short.nightly.tests.broad" refid="short.tests.broad" />`
right?

Comment (peterjcarr) [reply buildsuites.xml]: correct

File: buildsuites.xml (hunk 1 above)
Comment (bahill) [inline buildsuites.xml]: Line 268 - this doesn't impact the running of this - but you don't technically need Jenkins for this - just some sort of github/gpunit integration point.

File: buildsuites.xml (hunk 2 above)
Comment (bahill) [reply buildsuites.xml]: It should read:
`<pathconvert property="short.nightly.tests" refid="short.tests.broad" />`

See the IU test cases for example.

YOu can also just drop text2ODF from IU and Broad - we don't need to call out our not including them

Comment (bahill) [inline buildsuites.xml]: See my comment - sorry just getting used to review.

**Review by bahill — CHANGES_REQUESTED** [review]
Just line 314 needs updating and I would suggest removing the comments in the IU and Broad test sets regarding txt2ODF not being in there.

**Review by edjuaro — COMMENTED** [review]
I think this makes the changes that were suggested by Peter and Bobbie.

**Review by bahill — APPROVED** [review]
yep!

**Review by bahill — APPROVED** [review]
good to go

**Comment by bahill** [comment]
Approving Edwin's changes to add txt2odf to the new canon AWS tests (short-nightly) and new section for now archived Broad tests

**Pull request merged**